WCFDataService/Client/System/Data/Services/Client/MaterializeFromAtom.cs

```diff
@@ -69,6 +69,11 @@ internal class MaterializeAtom : IDisposable, IEnumerable, IEnumerator
         private bool moved;
 
         /// <summary>
+        /// The ODataResponseMessage to dispose when we are done.
+        /// </summary>
+        private IODataResponseMessage responseMessage;
```

Comment:
Yes, so before we may have body not disposed, and now we have the whole message. I agree that this can be reported as a separate issue.